### PR TITLE
Fixes in S3/Auth

### PR DESCRIPTION
**Pull request by ilblackdragon**
Remove double call to super in S3Provier and few error checks, replace input with raw_input in entering email.

### CODE REVIEW

File: studio/auth.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def __init__(
                 self.email = email
                 self.password = password
             else:
-                self.email = input(
+                self.email = raw_input(
```
Comment (pzhokhov) [inline studio/auth.py]: I think this should be six.input() actually for python 2/3 compatibility

Comment (pzhokhov) [reply studio/auth.py]: on the second thought, this file is soon to go entirely when we switch to github authentication, so don't worry about it

File: studio/s3_provider.py [hunk 2]
```diff
@@ -26,12 +19,22 @@ def __init__(self, config, blocking_auth=True, verbose=10, store=None):
...
     def _get(self, key, shallow=False):
-        response = self.meta_store.client.get_object(
-            Bucket=self.bucket,
-            Key=key)
+        try:
+            response = self.meta_store.client.get_object(
+                Bucket=self.bucket,
+                Key=key)
+        except self.meta_store.client.exceptions.NoSuchKey as e:
+            logging.error("Failed to retrieve key %s from %s", key, self.bucket)
```
Comment (pzhokhov) [inline studio/s3_provider.py]: this is a fairly routine situation - trying to access the key and realizing it does not exsit (esp. given that we have no separate method for detecting if key is there), so maybe do logger.info / logger.debug instead in order not to blow up the output

**Comment by pzhokhov** [comment]
Looks like the pep8 test is failing

**Comment by ilblackdragon** [comment]
Not relevany anymore.

**Pull request closed without merging**